core/store/src/trie/trie_storage.rs

```diff
@@ -116,7 +123,7 @@ impl TrieCacheInner {
                 .with_label_values(&metrics_labels),
         };
         Self {
-            cache: LruCache::new(cache_capacity),
+            cache: LruCache::new(cache_capacity as usize),
```

Comment:
Naively, I'd thought that we need 

https://docs.rs/lru/latest/lru/struct.LruCache.html#method.unbounded

here

Reply:
yes, that makes sense!